docs/user/integration/sample-app/README.md

```diff
@@ -0,0 +1,122 @@
+# Integrate Custom Instrumentation App
+
+## Overview
+
+| Category| |
+| - | - |
+| Signal types | traces, metrics, logs |
+| Backend type | custom in-cluster, third-party remote |
+| OTLP-native | yes |
+
+Learn how to instrument your custom Golang application using the [OTel SDK](https://opentelemetry.io/docs/languages/) and exporting metrics and trace data using the OTel SDK. The sample application will be configured to push trace and metric data using OTLP to the collector that's provided by Kyma, so that they are collected together with the related Istio trace data.
+
+For examples using the OTel-SDK in a different langue, please refer to the official [OTel guides](https://opentelemetry.io/docs/languages/) and the [OTel demo app](./../opentelemetry-demo/).
+
+![setup](./../assets/sample-app.drawio.svg)
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- The [Telemetry module](../../README.md) is [added](https://kyma-project.io/#/02-get-started/01-quick-install)
+- [Kubectl version that is within one minor version (older or newer) of `kube-apiserver`](https://kubernetes.io/releases/version-skew-policy/#kubectl)
+
+## Exploring the Sample App
+
+The sample app is a small webserver written in Golang, which exposes two endpoints `forward` and `terminate`. When calling the endpoints via HTTP, metrics are getting counted up and spans are emitted using OTLP exporters. Furthermore, structured logs are written to `stdout`.
+
+<!-- markdown-link-check-disable-next-line -->
+The application is located in the [`telemetry-manager`](https://github.com/kyma-project/telemetry-manager/tree/main/docs/user/integration/sample-app) repository.
+
+
+### Setup
+
+The application consists of following go files:
+- `main.go`
+    - the main method with the actual handler routines
+    - the initialization of the tracer and meter provider including the metric definitions
+    - auto-instrumentation of the handler routines using the OTEL-SDK
+- `setup.go`
+    - the setup of the tracer and meter providers using the OTel SDK
+    - configures either OTLP GRPC exporters or a console exporter
+
+### Metrics
+
+- The `init` method defines the available metrics
+  - `cpu.temperature.celsius` is a Gauge which gets updated constantly via an observable
+  - `hd.errors.total` is a Counter which gets increased on every `terminate` handler call
+  - `cpu.energy.watt` is a Histogram which gets increased in one bucket on every `terminate` handler call
+- The `main` method initializes and registers the meterProvider

```

Comment:
Is meterProvider a resource and that's why it's in camel case; or should it simply have a blank space?

```suggestion
- The `main` method initializes and registers the MeterProvider.
```

or 

```suggestion
- The `main` method initializes and registers the meter provider
```